.github/workflows/release.yml

```diff
@@ -26,7 +26,6 @@ jobs:
       - name: Use Node.js 18
         uses: actions/setup-node@v3
         with:
-          version: 9.0.6
```

Comment:
Nothing should change as a result of this removal, since `version: 9.0.6` has no effect on this action to begin with.

It should remove noise in the codebase, though.